fieldline_client/lib.py

```diff
@@ -1,46 +1,45 @@
 # mne_fieldline_connector
 # gabrielbmotta, juangpc
 
-import fieldline_client.mne_fieldline_config as config
-# import mne_fieldline_tools as tools
-
 import queue
 import time
 import threading
-import numpy
 
-import fieldline_client.FieldTrip as FieldTrip
+import numpy as np
+
+from .FieldTrip import Client, DATATYPE_FLOAT32
+from .config import (working_chassis, broken_sensors,
+                     working_sensors, ip_list,
+                     use_phantom, ft_IP, ft_port)
```

Comment:
order of package imports should be more general to specific. First Python libraries, then scientific libraries, then the files of this package